docs/customizations/static-etcd-configuration.md

```diff
@@ -0,0 +1,5 @@
+### Static etcd configuration
+
+In order to configure a static etcd cluster to scrape there is a simple [static-etcd.libsonnet](../../jsonnet/kube-prometheus/addons/static-etcd.libsonnet) mixin prepared - see [etcd.jsonnet](../../examples/etcd.jsonnet) for an example of how to use that mixin, and [Monitoring external etcd](../monitoring-external-etcd.md) for more information.
```

Comment:
Could you make it multiline? Also do we want to embed cited resources here for better readibility?